### PR TITLE
feat(docs): add quickstart guide & README

### PR DESCRIPTION
**Pull request by michelleN**
taking a stab at re-vamping docs #742

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+## Prerequisites
+
+The following prerequisites are required for successful use of Draft.
+
+1. A Kubernetes cluster and the `kubectl` CLI tool
+2. Installation of Helm on your Kubernetes cluster
+3. Installing and configuring Draft on your laptop
+
+### Install Kubernetes or have access to a cluster
+- If you don't already have a running Kubernetes cluster, consider downloading and installing [Minikube][] which can help you get a Kubernetes cluster running on your local machine
```
Comment (radu-matei) [inline docs/quickstart.md]: Should we consider also adding Docker for Mac/Windows here, as the Kubernetes feature will probaly get out of the beta channel around DockerCon?

I will test all the steps here with Docker for Mac and maybe add a PR after we know everything works.

File: docs/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+### Install and configure Helm
+- Helm is a package manager for Kubernetes. It allows you to install and manage packages of Kubernetes manifests, more commonly known as "charts", on your Kubernetes cluster.
+- Download the Helm binary using [Homebrew](https://brew.sh/) via `brew install kubernetes-helm` or the [official releases page](https://github.com/kubernetes/helm/releases).
+- Once you have Helm on your machine and a running Kubernetes cluster, run the following command:
+```console
+$ helm init
+```
+
+### Install and configure Draft
```
Comment (radu-matei) [inline docs/quickstart.md]: @bacongobbler - should we add Fish instructions for Draft?

File: docs/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+```console
+$ brew tap azure/draft
+$ brew install draft
+```
+- Set up Draft on your machine by running:
+```console
+$ draft init
+```
+
+Congratulations! You're all set! Check out the [Getting Started](getting-started.md) page to see how to use Draft with a sample application.
```
Comment (radu-matei) [inline docs/quickstart.md]: Before linking to the Getting Started guide, which starts with a `draft create` and `draft up`, should we first point to an installation guide? Or add the info here on how to config the registry / use the correct Docker context for Minikube?

File: docs/quickstart.md (hunk 2 above)
Comment (bacongobbler) [reply docs/quickstart.md]: Yes; I'd love to.

Comment (palma21) [reply docs/quickstart.md]: Maybe chocolatey as well? Especially for windows

Comment (bacongobbler) [reply docs/quickstart.md]: Theres no chocolatey package available for Draft. We're tracking that in another issue, but once that's available we can add that to the list as well.

Comment (bacongobbler) [reply docs/quickstart.md]: I will note however that GoFish works on Mac/Linux/Windows so it's something Windows users can try out.

@palma21 for reference: https://gofish.sh

Comment (palma21) [reply docs/quickstart.md]: Yes I also use gofish but just stumbled on https://chocolatey.org/packages/draft hence the comment

File: docs/quickstart.md (hunk 3 above)
Comment (michelleN) [reply docs/quickstart.md]: oh good catch! I added that part in based on your feedback. Thanks

**Pull request merged**